README.md

```diff
@@ -16,37 +16,55 @@ Please try the new version, and leave feedback, as we endeavour to improve our e
 
-> The extension is based on and/or inspired by : 
-> 
+
+# Acknowledgements
+The extension is based on and/or inspired by :
+
 > - **Original radar chart extension**, developed by **Borja Rodriguez** :  [here](http://bit.ly/QlikSenseRadarChart_by_BrianMunz)
 > - **D3.js**, using by **Nadieh Bremer** to design alternative radar chart look & feel :  [here](http://bit.ly/AlternativeRadarChartLookAndFeel_by_NadiehBremer)
 > - **Susie Lu's legend maker** [here](http://bit.ly/D3LegendJS_by_SusieLu)
 > - **Chroma.js**, **Gregor Aisch**'s color palette generator : [here](http://bit.ly/ChromaJS_by_GregorAisch)
 
-----------
 
-Distributed using MIT License
+# Installation
+
+1. Download the extension zip, `qlik-radar-chart_<version>.zip`, from the latest release(https://github.com/qlik-oss/D3ImprovedRadarChart/releases/latest)
+2. Install the extension:
+
+   a. **Qlik Sense Desktop**: unzip to a directory under [My Documents]/Qlik/Sense/Extensions.
+
+   b. **Qlik Sense Server**: import the zip file in the QMC.
+
+
+# Developing the extension
+
+If you want to do code changes to the extension follow these simple steps to get going.
+
+1. Get Qlik Sense Desktop
+1. Create a new app and add the extension to a sheet.
+2. Clone the repository
+3. Run `npm install`
+4. Set the environment variable `BUILD_PATH` to your extensions directory. It will be something like `C:/Users/<user>/Documents/Qlik/Sense/Extensions/<extension_name>`.
+5. You now have two options. Either run the watch task or the build task. They are explained below. Both of them default to development mode but can be run in production by setting `NODE_ENV=production` before running the npm task.
+
+   a. **Watch**: `npm run watch`. This will start a watcher which will rebuild the extension and output all needed files to the `buildFolder` for each code change you make. See your changes directly in your Qlik Sense app.
+
+   b. **Build**: `npm run build`. If you want to build the extension package. The output zip-file can be found in the `buildFolder`.
+
+
+# Original authors
+
+[github.com/brianbooden](https://github.com/brianbooden/)
+[https://github.com/VizMatt](https://github.com/VizMatt)
+
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+# License
 
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+Copyright © 2015 Brian Booden & Matthieu Burel
```

Comment:
License file has both original authors and qlik as copyright text, possibly just say MIT with a link to the license file?

Reply:
Fixed